backend/routes/person.py

```diff
@@ -18,4 +18,4 @@ def strip(self):
 
 
 def person_from_db_row(row) -> Person:
-    return Person(person_id=row[0], is_roow=row[1], nickname=row[2], session_cookie=row[3], mail=row[4], password=row[5])
+    return Person(person_id=row[0], is_root=row[1], nickname=row[2], session_cookie=row[3], mail=row[4], password=row[5])
```

Comment:
alredy fixed

Reply:
to je ok, mělo by se ztratit po updatu z `dev`